core/capabilities/syncer.go

```diff
@@ -143,38 +170,49 @@ func (s *registrySyncer) Start(ctx context.Context) error {
 			s.subServices = append(s.subServices, localTrigger)
 			s.subServices = append(s.subServices, mockMercuryDataProducer)
 			// ----
-		}
+		}*/
 
-		underlying, err2 := s.registry.GetTrigger(ctx, capId)
-		if err2 != nil {
-			// NOTE: it's possible that the jobs are not launched yet at this moment.
-			// If not found yet, Syncer won't add to Registry but retry on the next tick.
-			return err2
-		}
-		workflowDONs := map[string]capabilities.DON{
-			workflowDonInfo.ID: workflowDonInfo,
-		}
-		triggerCap := remote.NewTriggerPublisher(config, underlying, triggerInfo, triggerCapabilityDonInfo, workflowDONs, s.dispatcher, s.lggr)
-		err = s.dispatcher.SetReceiver(capId, triggerCapabilityDonInfo.ID, triggerCap)
-		if err != nil {
-			s.lggr.Errorw("capability DON failed to set receiver", "capabilityId", capId, "donId", triggerCapabilityDonInfo.ID, "error", err)
-			return err
+		count := 0
+		for {
+			count++
+			if count > maxRetrySeconds {
```

Comment:
Seems confusingly named, count doesn't appear to be a timer

Reply:
true, will remove the "seconds" thanks